web-ng/src/app/components/feature-panel/feature-panel.component.html

```diff
@@ -29,9 +31,13 @@
       </mat-card-title>
       <!--TODO: figure out why serverTime is not working-->
       <mat-card-subtitle>
-        {{ observation.created.clientTime | date:'medium' }}
+        {{ observation.created.clientTime | date: 'medium' }}
       </mat-card-subtitle>
-      <button mat-icon-button [matMenuTriggerFor]="menu" aria-label="actions">
+      <button
```

Comment:
Looks like you reverted the change that adds the overflow menu (three dots). I assume this wasn't intentional?

Reply:
It was intentional for testing this feature, fixed it. ty :)